### PR TITLE
expose node names

### PR DESCRIPTION
**Pull request by Karsten1987**
Connect to ros2/cli_tools#1

### CODE REVIEW

File: rmw_opensplice_cpp/src/rmw_node_names.cpp [hunk 1]
```diff
@@ -0,0 +1,47 @@
+// Copyright 2014-2015 Open Source Robotics Foundation, Inc.
```
Comment (dirk-thomas) [inline rmw_opensplice_cpp/src/rmw_node_names.cpp]: Year.

File: rmw_opensplice_cpp/src/rmw_node_names.cpp [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+#include <string>
+
+#include "rmw/allocators.h"
+#include "rmw/error_handling.h"
+#include "rmw/rmw.h"
+#include "rmw/sanity_checks.h"
+#include "rmw/types.h"
+
+#include "identifier.hpp"
+#include "types.hpp"
```
Comment (dirk-thomas) [inline rmw_opensplice_cpp/src/rmw_node_names.cpp]: Many of the includes are not used atm.

**Comment by Karsten1987** [comment]
rebased and updated.

File: rmw_opensplice_cpp/src/rmw_node_names.cpp (hunk 1 above)
Comment (dirk-thomas) [inline rmw_opensplice_cpp/src/rmw_node_names.cpp]: You actually imlemented this over a month ago?

**Comment by mikaelarguedas** [comment]
See https://github.com/ros2/rmw_opensplice/commit/885526a7fd7e3dfaf271486033f83cf7a7903746 for follow-up fixes

**Pull request merged**